pkg/athena/api/api.go

```diff
@@ -14,7 +14,7 @@ import (
 	sqlModels "github.com/grafana/grafana-aws-sdk/pkg/sql/models"
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 	sdkhttpclient "github.com/grafana/grafana-plugin-sdk-go/backend/httpclient"
-	"github.com/grafana/sqlds/v2"
+	"github.com/grafana/sqlds/v3"
```

Comment:
With the bump, the linter is complaining about deprecation warnings. The simplest way to fix that is to add a `.golangci.yml` file with the following contents:
```yaml
issues:
  exclude-rules:
    - path: ^pkg/athena/macros.go
      text: 'SA1019: sqlds.Query is deprecated'
    - path: ^pkg/athena/macros_test.go
      text: 'SA1019: sqlds.Query is deprecated'
```

Reply:
I know that the main repo uses `// nolint:staticcheck` comments above the line with the issue, and that might work here?

Reply:
That does also work, although there are 27 references to `sqlds.Query`, which might be a little annoying 😁

Ah, but maybe even better would be just replacing the deprecated sqlds types (Query, Macros, MacroFunc) with their sqlutil versions?

Reply:
I think I successfully switched the the sqlutil versions but let me know if I missed any!